covid19-etl/tests/test_data/test_cityofchicago_input.csv

```diff
@@ -1,11 +1,11 @@
 "lab_report_date","cases_total","deaths_total","hospitalizations_total","cases_age_0_17","cases_age_18_29","cases_age_30_39","cases_age_40_49","cases_age_50_59","cases_age_60_69","cases_age_70_79","cases_age_80_","cases_age_unknown","cases_female","cases_male","cases_unknown_gender","cases_latinx","cases_asian_non_latinx","cases_black_non_latinx","cases_white_non_latinx","cases_other_non_latinx","cases_unknown_race_eth","deaths_0_17_yrs","deaths_18_29_yrs","deaths_30_39_yrs","deaths_40_49_yrs","deaths_50_59_yrs","deaths_60_69_yrs","deaths_70_79_yrs","deaths_80_yrs","deaths_unknown_age","deaths_female","deaths_male","deaths_unknown_gender","deaths_latinx","deaths_asian_non_latinx","deaths_black_non_latinx","deaths_white_non_latinx","deaths_other_non_latinx","deaths_unknown_race_eth","hospitalizations_age_0_17","hospitalizations_age_18_29","hospitalizations_age_30_39","hospitalizations_age_40_49","hospitalizations_age_50_59","hospitalizations_age_60_69","hospitalizations_age_70_79","hospitalizations_age_80_","hospitalizations_age_unknown","hospitalizations_female","hospitalizations_male","hospitalizations_unknown_gender","hospitalizations_latinx","hospitalizations_asian_non_latinx","hospitalizations_black_non_latinx","hospitalizations_white_non_latinx","hospitalizations_other_race_non_latinx","hospitalizations_unknown_race_ethnicity"
 "2022-04-29T00:00:00.000","7","0",,"2","0","0","1","0","1","2","1","0","6","1","0","2","1","2","2","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0",,,,,,,,,,,,,,,,,,
-"2022-04-28T00:00:00.000","340","0",,"62","76","64","55","32","24","19","8","0","198","139","3","70","35","52","121","16","46","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0",,,,,,,,,,,,,,,,,,
+"2022-04-28T00:00:00.000",,"0",,"62","76","64","55","32","24","19","8","0","198","139","3","70","35","52","121","16","46","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0",,,,,,,,,,,,,,,,,,
```

Comment:
So I removed some values in the file to make edge tests. Apart from hospitalization data, if any data is missing, it is replaced by zero. currently in the original dataset only hospitalization data is updated later after the day passed so we are using our logic to get data later but I haven't put condition for cases and deaths column.

Reply:
ok, i have the same concern here, i think we should error out if this happens 🤔 have you ever seen this case in the real dataset?

Reply:
No. I haven't.

Reply:
i would say the ETL should not accept this either

Reply:
> or it should error out for missing cases and deaths values and use last_submission_identifier only for missing hospitalization records?

i would choose this so we don't over-engineer, since we don't expect them to update cases and deaths, but your call

Reply:
I agree, I committed the code to raise exception if cases and deaths data is not found.